packages/@sanity/vision/src/components/VisionGui.tsx

```diff
@@ -705,6 +705,7 @@ export class VisionGui extends React.PureComponent<VisionGuiProps, VisionGuiStat
           </Grid>
         </Header>
         <SplitpaneContainer flex={1}>
+          {/* @ts-expect-error: https://github.com/tomkp/react-split-pane/pull/819 */}
```

Comment:
Follow up in https://app.shortcut.com/sanity-io/story/25140/replace-react-split-pane-with-allotment